src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/IdentifiersShouldNotHaveIncorrectSuffix.cs

```diff
@@ -147,12 +149,17 @@ public override void Initialize(AnalysisContext analysisContext)
                                 return;
                             }
 
+                            var allowedSuffixes = namedTypeSymbol.DeclaringSyntaxReferences
+                                .Select(syntaxRef => symbolAnalysisContext.Options.GetStringOptionValue(EditorConfigOptionNames.AllowedSuffixes, TypeNoAlternateRule, syntaxRef.SyntaxTree, symbolAnalysisContext.Compilation, symbolAnalysisContext.CancellationToken))
```

Comment:
I was thinking about returning `ImmutableHashSet<string>` instead of string but that doesn't seem to bring much.